coriolis/providers/replicator.py

```diff
@@ -440,6 +440,22 @@ def wait_for_chunks(self):
             else:
                 time.sleep(5)
 
+    def start(self):
+        utils.start_service(self._ssh, REPLICATOR_SVC_NAME)
+
+    def stop(self):
+        utils.stop_service(self._ssh, REPLICATOR_SVC_NAME)
+
+    def restart(self):
+        utils.restart_service(self._ssh, REPLICATOR_SVC_NAME)
+
+    def update_state(self, state, restart=False):
+        state_file = self._get_replicator_state_file()
```

Comment:
This should be done explicitly from outside the replicator, and passed in using the ```state``` parameter. The idea is to allow the person using this function, to supply whichever state, previous or otherwise, they see fit.

The ```state``` parameter should be a ```json``` or a serializable python object that when serialized to json, results in a valid replicator state file. Your choice.